### PR TITLE
Added os to PlatformInfo

### PR DESCRIPTION
**Pull request by beckyconning**

### CODE REVIEW

File: src/Platform.purs [hunk 1]
```diff
@@ -21,6 +21,7 @@ type PlatformInfo =
   , prerelease :: String
   , product :: String
   , ua :: String
+  , os :: { architecture :: Int, family :: String }
```
Comment (cryogenian) [inline src/Platform.purs:24]: Please, add `version` field to `os`.

Comment (beckyconning) [inline src/Platform.purs:24]: What type would it be? `Maybe String`?

Comment (cryogenian) [inline src/Platform.purs:24]: `Maybe String` is ok.

**Comment by cryogenian** [comment]
This one is already implemented

**Pull request closed without merging**